extensions/convert-3d-models/package.json

```diff
@@ -0,0 +1,228 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "convert-3d-models",
+  "title": "Convert 3D Models",
+  "description": "An extension to convert 3D models to different formats.",
+  "icon": "extension_icon.png",
+  "author": "mrnoisytiger",
+  "categories": [
+    "Design Tools",
+    "Productivity",
+    "Other"
+  ],
+  "license": "MIT",
+  "preferences": [
+    {
+      "title": "FreeCAD Path",
+      "name": "freeCADPath",
+      "description": "Path where FreeCAD is installed (with the trailing slash). Typically /Applications/FreeCAD.app/.",
+      "type": "textfield",
```

Comment:
```suggestion
      "type": "appPicker",
```
Could you see if that would work for you 🙂

Reply:
@pernielsentikaer of course that exists! Perfect, thank you!